### DISCUSSION TITLE
PRIMEBLOCKS' developer

### DISCUSSION THREAD

**Discussion by AbdouAzizDIOUF** [primefaces/community#1732] [PrimeNG Blocks] [answered]
I just bought a 'PRIMEBLOCKS' developer license from PrimeNG, but it turns out it's not available for Angular versions 17/16. Could you please provide me with more details?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

PrimeBlocks are copy-paste elements, you can use it in v17 and v16.
